<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Streaming</class>
 <widget class="QWidget" name="Streaming">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="streamingFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border:none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <widget class="QLabel" name="streamingLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>671</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 300 bold 13pt &quot;Quicksand Light&quot;;
color:  rgb(243,115,33);
background:rgba(243,115,33,0.2);
border:1px solid rgb(243,115,33);</string>
    </property>
    <property name="text">
     <string>Streaming Video</string>
    </property>
   </widget>
   <widget class="QFrame" name="bestshotFrame">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>290</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 13pt &quot;Quicksand Medium&quot;;
color:  rgb(243,115,33);
background:none;
border:1px solid rgb(243,115,33);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:none;
border:1px solid rgb(255, 255, 255);
font: 300 bold 13pt &quot;Quicksand Light&quot;;
color:  rgb(243,115,33);
background:none;</string>
     </property>
     <property name="text">
      <string>Detected Vehicle Image</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:none;
border:1px solid rgb(255, 255, 255);
font: 300 bold 13pt &quot;Quicksand Light&quot;;
color:  rgb(243,115,33);
background:none;</string>
     </property>
     <property name="text">
      <string>Vehicle1</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="dateTimeLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 500 12pt &quot;Quicksand Medium&quot;;
background-color: rgba(243,115,33,0.5);
border:1px solid rgb(243,115,33);
text-align: center; /* 수평 중앙 정렬 */</string>
    </property>
    <property name="text">
     <string>Date - Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="licensetitleLabel">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>290</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 500 13pt &quot;Quicksand Medium&quot;;
background-color: rgba(243,115,33,0.5);
border:1px solid rgb(243,115,33);</string>
    </property>
    <property name="text">
     <string>LicensePlate</string>
    </property>
   </widget>
   <widget class="QFrame" name="bestshotFrame_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(243,115,33);
background:none;
border:1px solid rgb(243,115,33);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color:  rgb(243,115,33);
	background:white;
	border:1px solid rgb(243,115,33);
	font: 500 10pt &quot;Quicksand Medium&quot;;
}

QPushButton:hover {
	color:  rgb(255, 255, 255);
	background-color:rgba(243,115,33,0.3);
	border:1px solid rgb(243,115,33);
	font: 500 8pt &quot;Quicksand Medium&quot;;
}</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color:  rgb(243,115,33);
	background:white;
	border:1px solid rgb(243,115,33);
	font: 500 10pt &quot;Quicksand Medium&quot;;
}

QPushButton:hover {
	color:  rgb(255, 255, 255);
	background-color:rgba(243,115,33,0.3);
	border:1px solid rgb(243,115,33);
	font: 500 8pt &quot;Quicksand Medium&quot;;
}</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_url">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
	color:  rgb(243,115,33);
	background:white;
	border:1px solid rgb(243,115,33);
	font: 500 10pt &quot;Quicksand Medium&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>streamingLabel</zorder>
   <zorder>dateTimeLabel</zorder>
   <zorder>licensetitleLabel</zorder>
   <zorder>bestshotFrame</zorder>
   <zorder>bestshotFrame_2</zorder>
   <zorder>startButton</zorder>
   <zorder>stopButton</zorder>
   <zorder>label_url</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
